Source/WebCore/css/CSSToStyleMap.h

```diff
@@ -39,55 +36,55 @@ class Quad;
     void mapFillXPosition(CSSPropertyID, FillLayer&, const CSSValue&);
     void mapFillYPosition(CSSPropertyID, FillLayer&, const CSSValue&);
-    void mapFillMaskMode(CSSPropertyID, FillLayer&, const CSSValue&);
+    static void mapFillMaskMode(CSSPropertyID, FillLayer&, const CSSValue&);
 
-    void mapAnimationDelay(Animation&, const CSSValue&);
-    void mapAnimationDirection(Animation&, const CSSValue&);
-    void mapAnimationDuration(Animation&, const CSSValue&);
-    void mapAnimationFillMode(Animation&, const CSSValue&);
-    void mapAnimationIterationCount(Animation&, const CSSValue&);
+    static void mapAnimationDelay(Animation&, const CSSValue&);
+    static void mapAnimationDirection(Animation&, const CSSValue&);
+    static void mapAnimationDuration(Animation&, const CSSValue&);
+    static void mapAnimationFillMode(Animation&, const CSSValue&);
+    static void mapAnimationIterationCount(Animation&, const CSSValue&);
     void mapAnimationName(Animation&, const CSSValue&);
-    void mapAnimationPlayState(Animation&, const CSSValue&);
-    void mapAnimationProperty(Animation&, const CSSValue&);
-    void mapAnimationTimingFunction(Animation&, const CSSValue&);
-    void mapAnimationCompositeOperation(Animation&, const CSSValue&);
+    static void mapAnimationPlayState(Animation&, const CSSValue&);
+    static void mapAnimationProperty(Animation&, const CSSValue&);
+    static void mapAnimationTimingFunction(Animation&, const CSSValue&);
+    static void mapAnimationCompositeOperation(Animation&, const CSSValue&);
 
-    void mapNinePieceImage(CSSValue*, NinePieceImage&);
-    void mapNinePieceImageSlice(CSSValue&, NinePieceImage&);
-    void mapNinePieceImageSlice(CSSBorderImageSliceValue&, NinePieceImage&);
-    void mapNinePieceImageWidth(CSSValue&, NinePieceImage&);
-    void mapNinePieceImageWidth(CSSBorderImageWidthValue&, NinePieceImage&);
-    LengthBox mapNinePieceImageQuad(CSSValue&);
-    void mapNinePieceImageRepeat(CSSValue&, NinePieceImage&);
-    void mapNinePieceImageRepeat(CSSPrimitiveValue&, NinePieceImage&);
+    void mapNinePieceImage(const CSSValue*, NinePieceImage&);
+    static void mapNinePieceImageSlice(const CSSValue&, NinePieceImage&);
+    static void mapNinePieceImageSlice(const CSSBorderImageSliceValue&, NinePieceImage&);
+    void mapNinePieceImageWidth(const CSSValue&, NinePieceImage&);
+    void mapNinePieceImageWidth(const CSSBorderImageWidthValue&, NinePieceImage&);
+    LengthBox mapNinePieceImageQuad(const CSSValue&);
+    static void mapNinePieceImageRepeat(const CSSValue&, NinePieceImage&);
 
 private:
     RenderStyle* style() const;
-    RefPtr<StyleImage> styleImage(CSSValue&);
-    LengthBox mapNinePieceImageQuad(Quad&);
+    RefPtr<StyleImage> styleImage(const CSSValue&);
+    LengthBox mapNinePieceImageQuad(const Quad&);
+    Length mapNinePieceImageSide(const CSSValue&);
 
-    // FIXME: This type can merge into BuilderState.
```

Comment:
These are a bunch of helpful functions that are used for style building. Not sure they should be called a "map" just because they map styles. Organizing them as a class is OK, but they could also just be non-member functions and some would take `BuilderState&` arguments. And, yes, they could be `BuilderState` member functions.